input/fsh/at-core-patient.fsh

```diff
@@ -55,3 +55,16 @@ Description:    "HL7® Austria FHIR® Core Profile for patient pata in Austria."
 
 //Extension for Religion -> HL7 Austria's AustrianReligion Extension
 * extension contains PatientReligion named PatientReligion 0..1
+
+
+Mapping:  HL7ATCorePatient2CdaATv3
+Source:   at-core-patient
+Target:   "https://wiki.hl7.at/index.php%3ftitle=ILF:Allgemeiner_Implementierungsleitfaden_(Version_3)"

```

Comment:
maybe try to package it into  <a href="https://wiki.hl7.at/index.php%3ftitle=ILF:Allgemeiner_Implementierungsleitfaden_(Version_3)">HL7 Allgemeiner Implementierungsleitfaden CDA</a>

Reply:
Tried short link : 
![image](https://user-images.githubusercontent.com/63285594/150340476-754f268f-d852-4a17-be83-34270ee3dcba.png)

Tried in <a>-Element
![image](https://user-images.githubusercontent.com/63285594/150341047-c67638ef-26cf-4bd5-a549-820aa3a31d47.png)

none worked.

No href is possible

Reply:
Will be documented as an known issue, seems like a sushi problem: https://github.com/HL7/fhir/blob/10c1d1796ac49c9552adf0423845df307639a472/source/supplyrequest/structuredefinition-SupplyRequest.xml#L13